src/sentry/static/sentry/app/components/smartSearchBar/index.jsx

```diff
@@ -446,6 +447,38 @@ class SmartSearchBar extends React.Component {
     ];
   };
 
+  getReleases = _.debounce(
+    async (tag, query) => {
+      const releasePromise = this.fetchReleases(query);
+
+      const tags = this.getPredefinedTagValues(tag, query);
+      const tagValues = tags.map(v => ({
+        ...v,
+        type: 'first-release',
+      }));
+
+      const releases = await releasePromise;
+      const releaseValues = releases.map(r => ({
+        value: r.shortVersion,
+        desc: r.shortVersion,
+        type: 'first-release',
+      }));
+
+      return [...tagValues, ...releaseValues];
+    },
+    300,
```

Comment:
I'm likely butting my head into something I don't know too well, but is it worthwhile to push the debounce value into the constants as well?

Reply:
Fixed: added `DEBOUNCE_DURATION` into constants.

Reply:
Fixed: added `MAX_AUTOCOMPLETE_RELEASES` into constants file. 

IMO the global constants file is more appropriate for it because there are a few SmartSearchBar constants that are used somewhere else on the codebase too. It is much cleaner to have a single source file for constants (as compared to importing them as needed from random components).